modules/statistics/php/statistics_dd_site.class.inc

```diff
@@ -25,9 +25,49 @@ namespace LORIS\statistics;
  */
 class Statistics_DD_Site extends statistics_site
 {
-
     var $query_criteria = '';
     var $query_vars     = array();
+    /**
+     * Checking user's permission
+     *
+     * @param \User $user The user whose access is being checked
+     *
+     * @return bool
+     */
+    function _hasAccess(\User $user) : bool
+    {
+        // TODO (#6742): Create a permission specific to statistics
```

Comment:
I feel like this should not be in the code but rather should be documented in the PR. Same thing for all the other mentions of issues.